graphstorm-processing/graphstorm_processing/graph_loaders/schema_utils.py

```diff
@@ -92,9 +92,11 @@ def determine_spark_feature_type(feature_type: str) -> Type[DataType]:
         In case an unsupported feature_type is provided.
     """
     # TODO: Replace with pattern matching after moving to Python 3.10?
-    if feature_type in ["no-op", "multi-numerical"] or feature_type.startswith("text"):
+    if feature_type in ["no-op", "multi-numerical", "multi-categorical"] or feature_type.startswith(
+        "text"
+    ):
         return StringType
-    if feature_type in ["numerical", "bucket-numerical", "none"]:
+    if feature_type in ["numerical", "bucket-numerical", "none", "categorical"]:
```

Comment:
Categorical features should be parsed as `StringType` (text), move to the case above.